ONSdigital/design-system
# Update lint-staged and config

### What is the context of this PR?

Fixes: #2996 

This fixes the issue with linters not being run correctly on commit by updating `lint-staged` and updating the `lint-staged` config

### How to review this PR

- Checkout this branch
- Add some code to a js file that should be fixed by the linter such as "console.log("test");"
- Make a commit but don't push and notice that the line you added to the js file is automatically changed to "console.log('test');" using single quotes

### Checklist

This needs to be completed by the person raising the PR.
